android/src/main/java/com/geektime/reactnativeonesignal/RNOneSignal.java

```diff
@@ -201,6 +195,8 @@ public String getName() {
     @Override
     public void onHostDestroy() {
         OneSignal.removeNotificationOpenedHandler();
+
+        //Added to latest Android release
         OneSignal.removeNotificationReceivedHandler();
```

Comment:
When is the Android SDK plans to be updated?
I suggest waiting until we figure out all the errors here...